docs/data/charts/lines/lines.md

```diff
@@ -175,6 +175,20 @@ Different series could even have different interpolations.
 
 {{"demo": "InterpolationDemoNoSnap.js", "hideToolbar": true}}
 
+### Baseline
+
+The area chart draws a `baseline` on the Y axis `0`.
+This is useful as a base value, but customized visualizations may require a different baseline.
+
+To change it, simply provide the wanted `baseline` number of a series.
+It is also possible to provide `min` or `max` for the baseline to automatically follow the minimum or maximum value of the series.
+
+:::info
+Keep in mind the baseline only affects the area visuals. If you want to change the axis minimum or maximum values, you should use the `min/max` property of the axis.
+:::
```

Comment:
Not exactly if the baseline means we will ignore `0` for extremum computation

Reply:
I don't get what you mean here 🤔

Reply:
Sorry, I forgot a part of the sentence

If the baseline is defined with min/max, the 0 will be ignored for axis extremums

Reply:
Ah, this is an effect of your change, I didn't intend it at first. We can keep to the other discussion and I will update it with the outcomes